coderedcms/templates/coderedcms/pages/base.html

```diff
@@ -166,6 +166,7 @@
   {% block custom_scripts %}{% endblock %}
 
   {% include "wagtailseo/struct_data.html" %}
+  {% block struct_seo_extra %}{% endblock %}
```

Comment:
Perhaps it would make sense to put `{% include "wagtailseo/struct_data.html" %}` inside a block, to facilitate easy overriding. That would be backwards compatible with existing templates since it is a new block that they were not aware of.